modules/system/models/MailSetting.php

```diff
@@ -91,6 +99,10 @@ public static function applyConfigValues()
         $config->set('mail.from.address', $settings->sender_email);
 
         switch ($settings->send_mode) {
+            case self::MODE_FAILOVER:
+                $config->set('mail.mailers.failover.mailers', explode(',', $settings->failover_mailers) ?? ['log']);
```

Comment:
```suggestion
                $config->set('mail.mailers.failover.mailers', explode(',', $settings->failover_mailers));
```
`explode()` will never return `NULL`